openedx_events/enterprise/signals.py

```diff
@@ -0,0 +1,23 @@
+"""
+Custom signals definitions that representing the Open edX platform events.
+
+All signals defined in this module must follow the name and versioning
+conventions specified in docs/decisions/0002-events-naming-and-versioning.rst
+
+They also must comply with the payload definition specified in
+docs/decisions/0003-events-payload.rst
+"""
+
+from openedx_events.enterprise.data import LicenseLifecycle
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# .. event_type: org.openedx.enterprise.subscription.license.modified.v1
+# .. event_name: LICENSE_CREATED
+# .. event_description: emitted with Subscription.License is created.
```

Comment:
1. with => when?
2. Can we use plain english, rather than dotted notation (e.g. "Subscription License")?
3. Can you add to the description on its own line, something like: "WARNING: This event is being used for prototype purposes and should not be used, and should not be copied as a fully vetted pattern."
4.